settings/prettier/prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  jsxSingleQuote: true,
```

Comment:
Ah just noticed this - I think this is a pretty rare configuration since JSX is basically XML. Up to you but I think most projects use double quote in JSX.

Reply:
Isn't JSX not an XML? It allows things like `className={myClassName}`. I thought it's good for consistency because otherwise it would mean two different single quote rules in a single .jsx file. What do you think?

Reply:
Good point that it's not true XML. But I've never seen a project use this flag, including our two main dependencies material-ui and react. So I'd say it's not conventional to use jsxSingleQuote. Considering the quotes are used in totally different ways (const foo = 'bar' vs `<Component foo="bar">`, I guess it's rare to cause confusion.

Reply:
Removed `jsxSingleQuote: true` and reverted auto-fixes